src/roman_datamodels/datamodels.py

```diff
@@ -300,6 +303,57 @@ class GuidewindowModel(DataModel):
     pass
 
 
+class ModelContainerModel(DataModel, Sequence):
+    # Needs a number of methods to properly handle the contents
+    def __init__(self, init=None, asn_schema= None, asn_file_path=None, model_file_path=None, iscopy=False, **kwargs):
+        # __init__(self, init=None, asn_exptypes=None, asn_n_members=None,
+        #                  iscopy=False, **kwargs):
+        super().__init__(init=None, **kwargs)
+
+        self._models = []
+        self._iscopy = iscopy
+        # self.asn_exptypes = asn_exptypes
+        # self.asn_n_members = asn_n_members
+        # self.asn_table = {}
+        # self.asn_table_name = None
+        # self.asn_pool_name = None
+
+        self._memmap = kwargs.get("memmap", False)
```

Comment:
Memory mapping was disabled in jwst. My recollection is this was due to difficulties with FITS. Is it necessary to disable it for ASDF files?